CHANGELOG.md

```diff
@@ -1,6 +1,17 @@
 Changelog
 =========
 
+## [1.13.0] - 2019-04-01
+### Added
+ * Add MaxCloseTime as a graceful close timeout (#724)
```

Comment:
can we update to something like:
 * Add `MaxCloseTime` which sets a timeout for graceful connection close.

 - prefer backticks for go type/field names
 - more user-friendly explanation